lib/rock_rms/responses/donation.rb

```diff
@@ -1,22 +1,26 @@
 module RockRMS
-  class Donation
-    def self.format(response)
-      if response.is_a?(Array)
-        response.map { |donation| format_donation(donation) }
-      else
-        format_donation(response)
+  module Responses
```

Comment:
@taylorbrooks forgot to mention - any of these changes will require backwards compat update like:
```ruby
RockRMS::Donation = RockRMS::Responses::Donation
```
OR you will need to do a major version bump.

see https://github.com/taylorbrooks/rock_rms/commit/4748e297ceab0b448206d9afa6bde011a9102813#diff-89d48776cf7bcee18143c818f3d1258aR34

Reply:
This is a good point. Do you directly call RockRMS::Donation methods when you use the gem? In that situation this is a major breaking change (when I wrote it, I hadn't considered that people might use RockRMS::Donation directly)

Reply:
@jpowell meant to tag you in my previous comment question

Reply:
I don't, but someone else *could*.

Reply:
I don't think it's a breaking change. What good is calling a response formatter without the response?

Reply:
In open source, you can't assume that something obscure in the code base isn't being used by someone else just because you aren't using it. I'd be fine if the discussion was around a method signature of a *private* method. This change by definition is a breaking change... if we don't want to provide backwards compat references, then we bump the major version and call it a day. If we're against that, then provide backwards compat references and bump a minor version. Whichever route we go, it may make sense to make all the formatters consistent in one fell swoop.

I'll leave the direction to you @taylorbrooks. Just my 2¢